### PR TITLE
added Split operator

### PR DESCRIPTION
**Pull request by MissMeriel**
added Split tensor operator class

### CODE REVIEW

File: dnnv/nn/operations/tensor.py [hunk 1]
```diff
@@ -221,6 +221,22 @@ def from_onnx(cls, onnx_node, *inputs):
...
+    def __init__(self, x, axis, split, *, name: Optional[str] = None):
+        super().__init__(name=name)
+        self.x = x
+        self.axis = axis
+        self.split = split
+
+    @classmethod
+    def from_onnx(cls, onnx_node, *inputs):
+        attributes = {a.name: as_numpy(a) for a in onnx_node.attribute}
+        axis = attributes.get("axis")
```
Comment (dlshriver) [inline dnnv/nn/operations/tensor.py:234]: The axis attribute should have a default value.

```suggestion
        axis = attributes.get("axis", 0)
```

**Review by dlshriver — CHANGES_REQUESTED** [review]
This is a useful enhancement. To get this operation added, could you also add methods to the OnnxConverter and TensorflowConverter to support this operation, as well as add tests to the test suite to ensure that those methods work as expected?

File: dnnv/nn/operations/tensor.py (hunk 1 above)
Comment (MissMeriel) [reply dnnv/nn/operations/tensor.py]: Thanks! Updated with converter functionality, tests, and stylistic changes

**Review by dlshriver — APPROVED** [review]
LGTM 👍

**Pull request merged**